src/configuration/vscodeWorkspaceConfiguration.ts

```diff
@@ -143,6 +152,34 @@ export class VscodeWorkspaceConfiguration implements IWorkspaceConfiguration {
         return this.testExplorerConfiguration.get<string[]>('testplanArgs', []);
     }
 
+    private isBehaveTestEnabled(): boolean {
+        const overriddenTestFramework = this.testExplorerConfiguration.get<string | null>('testFramework', null);
+        if (overriddenTestFramework) {
+            return 'behave' === overriddenTestFramework;
+        }
+        return this.getConfigurationValueOrDefault(
+            this.pythonConfiguration,
+            ['testing.behaveEnabled', 'testing.BehaveEnabled'],
```

Comment:
I don't think you need `testing.BehaveEnabled`, for other test frameworks this was done for backward compatibility. Same for other functions here.